src/core/server/config/__tests__/config_service.test.ts

```diff
@@ -34,7 +35,9 @@ test('returns config at path as observable', async () => {
   const configService = new ConfigService(config$, defaultEnv, logger);
 
   const configs = configService.atPath('key', ExampleClassWithStringSchema);
-  const exampleConfig = await k$(configs)(first(), toPromise());
+  const exampleConfig = await from(configs)
```

Comment:
I think it's reasonable to expect that the observables from `configService.atPath()` are observables that have `.pipe()` and `.toPromise()` methods, so I don't think we need to use `from()` in all the places we're using it now.